integration-tests/tests/src/tests/realm-constructor.ts

```diff
@@ -24,6 +24,10 @@ import { IPerson, PersonSchema, DogSchema } from "../schemas/person-and-dogs";
 const RealmAsAny = Realm as any;
 
 describe("Realm#constructor", () => {
+    beforeEach(() => {
+        Realm.clearTestState();
+    });
+
```

Comment:
I like how this is being explicitly done now.  Good change! :)

Reply:
Yeah - it turned out we didn't want to do this after every test, if multiple tests are using the same open Realm.